pyerrors/fits.py

```diff
@@ -462,6 +462,17 @@ def _standard_fit(x, y, func, silent=False, **kwargs):
 
     if kwargs.get('correlated_fit') is True:
         corr = covariance(y, correlation=True, **kwargs)
+        
+        
+        if kwargs['cutoff']:
+            cutoff=kwargs.get('cutoff')
+            eigvals=np.linalg.eig(corr)[0]
+            eigvecs=np.array(np.linalg.eig(corr)[1:])[0]
+            max_eigval=np.max(np.abs(eigvals))
+            eigvals[np.abs(eigvals)< max_eigval*cutoff]=0
```

Comment:
I am not totally sure about the method, but as I understood it, you would like to set the inverse of the eigenvalues to zero if the eigenvalue is too large, right? If you set the eigenvalue to zero, the inversion below has to fail, I guess.